bricksrc/equipment.py

```diff
@@ -40,43 +40,43 @@
             "PlugStrip": {
                 "tags": [TAG.PlugStrip, TAG.Equipment],
             },
-            "Meter": {
-                "tags": [TAG.Meter, TAG.Equipment],
```

Comment:
I agree with moving `Meter` out of the Electrical subsystem hierarchy, but we might want to also eliminate the `Power_System` class (its referred to as the parent of some classes in `Meter` as well). It's not clear what the Power System of a building is, so I think we should remove until we have a better/clearer structure